contracts/Comptroller.sol

```diff
@@ -1336,6 +1368,33 @@ contract Comptroller is
         return _actionPaused[market][action];
     }
 
+    /**
+     * @notice Returns the assets an account has entered
+     * @param account The address of the account to pull assets for
+     * @return A list with the assets the account has entered
+     */
+    function getAssetsIn(address account) public view returns (VToken[] memory) {
+        uint256 len = 0;
+        for (uint256 i = 0; i < accountAssets[account].length; i++) {
+            Market storage market = markets[address(accountAssets[account][i])];
+            if (market.isListed) {
+                len++;
+            }
+        }
+
+        VToken[] memory assetsIn = new VToken[](len);
+        uint256 index = 0;
+        for (uint256 i = 0; i < accountAssets[account].length; i++) {
+            Market storage market = markets[address(accountAssets[account][i])];
+            if (market.isListed) {
+                assetsIn[index] = accountAssets[account][i];
+                index++;
+            }
+        }
+
```

Comment:
We can avoid initialization of the index, unchecked increment can be preffered

Reply:
maybe you misunderstood me, We can simply declare uint256 index; instead on initialization `uint256 index = 0` and `uint256 i = 0`, The default value would be 0 only.